web/react/stores/channel_store.jsx

```diff
@@ -270,4 +270,5 @@ ChannelStore.dispatchToken = AppDispatcher.register(function(payload) {
     }
 });
 
+ChannelStore.setMaxListeners(11);
```

Comment:
https://nodejs.org/api/events.html#events_emitter_setmaxlisteners_n

Apparently it's the perfect number for dealing with memory leaks